### PR TITLE
Extracting verbs for english corpus

### PR DESCRIPTION
**Pull request by anshulgo**
In english the verbs POS tags are VB, VBD, VBN, VBP, VBZ so the grep expression was modified to capture all the five.

### CODE REVIEW

**Comment by marfox** [comment]
Good point, thanks.
Can you please check if similar behaviors apply to other languages?
If so, can you implement grep expressions accordingly?

File: extract_verbs.sh [hunk 1]
```diff
@@ -114,7 +114,14 @@ find extracted -type f -exec cat {} \; > all-extracted.txt
 # Extract verbs with TreeTagger
 # N.B. treetagger segfaults with the single big file, run it over each article instead
 #cat all-extracted.txt | treetagger/cmd/tree-tagger-italian | grep VER | sort -u > verbi.txt
-find extracted -type f -exec bash -c "cat '{}' | treetagger/cmd/tree-tagger-"$LANGUAGE" | grep VER >> verbs" \;
+#For english corpus
+if [$LANGCODE == 'en']; then
```
Comment (marfox) [inline extract_verbs.sh]: I think we can make this clearer by moving the `verbTag` variable up in the switch statement which handles all the languages.

File: extract_verbs.sh [hunk 2]
```diff
@@ -114,7 +114,14 @@ find extracted -type f -exec cat {} \; > all-extracted.txt
...
+	# verbTag is regex grep expression for extracting all the four verb POS tags in english i.e VB, VBN, VBD,VBZ
+	verbTag='[[:space:]]VB[[:space:]]\|[[:space:]]VBZ[[:space:]]\|[[:space:]]VBD[[:space:]]\|[[:space:]]VBN[[:space:]]\|[[:space:]]VBP[[:space:]]'
```
Comment (marfox) [inline extract_verbs.sh]: Please prettify this regex.
If we just need `VB, VBD, VBN, VBP, or VBZ`, what do you think of just `VB[DNPZ]?`?

**Comment by anshulgo** [comment]
I have done all the fixes you suggested and added verb tags for languages which I was able to find over internet.
Please review.

File: extract_verbs.sh [hunk 3]
```diff
@@ -16,6 +16,7 @@ fi
 LANGUAGE="$(echo $1 | tr '[:upper:]' '[:lower:]')"
 
 # Switch statement to select language
+verbTag='[[:space:]]VER[[:space:]]'
```
Comment (marfox) [inline extract_verbs.sh]: No need for initialization here

File: extract_verbs.sh [hunk 4]
```diff
@@ -24,14 +25,19 @@ case $LANGUAGE in
 	"chinese" | "zh")
   		LANGUAGE="chinese"
   		LANGCODE="zh"
+  		verbTag='[[:space:]]V[ACEV][[:space:]]'
+
```
Comment (marfox) [inline extract_verbs.sh]: Extra line, please remove. This also applies below.

File: extract_verbs.sh (hunk 3 above)
Comment (marfox) [inline extract_verbs.sh]: The regex is not correct, there are no spaces around verb tags. Please remove `[[:space:]]`

File: extract_verbs.sh [hunk 5]
```diff
@@ -52,18 +60,25 @@ case $LANGUAGE in
 	"german" | "de")
   		LANGUAGE="german"
   		LANGCODE="de"
+  		verbTag='[[:space:]]PTKVZ[[:space:]]\[[:space:]]VVFIN[[:space:]]\[[:space:]]VAFIN[[:space:]]\[[:space:]]VMFIN[[:space:]]'
+
   		;;
 	"italian" | "it")
   		LANGUAGE="italian"
   		LANGCODE="it"
+  		verbTag='[[:space:]]V[AM]?[[:space:]]'
```
Comment (marfox) [inline extract_verbs.sh]: This is not the correct regex for Italian.

**Comment by marfox** [comment]
Hi @anshulgo, all your regexes contain `[:space:]` and won't work.
I didn't check for other languages, but for Italian the verb tag is also completely wrong.
This suggests you didn't test your regexes at all.
Also, some languages are missing.

**Pull request closed without merging**